app/uk/gov/hmrc/trackingconsentfrontend/assets/src/interfaces/getMessages.ts

```diff
@@ -5,7 +5,7 @@ import messagesCy from '../../../../../../../../conf/messages.cy';
 
 import getLanguage from "./getLanguage";
 import {CY} from "../constants/languages";
-import getWelshLanguageEnabled from "./getWelshLanguageEnabled";
+import { getWelshLanguageEnabled } from "../config/config";
```

Comment:
`config/config` doesn't feel like a great path - can we use `./config` or something more specific instead of the second `config`?

Reply:
I've moved these functions into /common for now.